spec/mastermind/random_ai_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+module Mastermind
+
+  describe RandomAI do
+
+    it "returns a random guess that is 4 characters long" do
+      ai = RandomAI.new
+      expect(ai.next_guess(0, 0).length).to eq(4)
+    end
+
+    it "returns a guess and each character is one of RGOYBP" do
+      ai = RandomAI.new
+      guess = ai.next_guess(0, 0)
+
+      guess.each_char do |char|
+        expect(Guess::COLOURS).to include(char)
+      end
```

Comment:
I'm not too sure how to go about doing TDD to check that the guess has a random values from an array for it's 4 characters.